core/spec.md

```diff
@@ -4035,13 +4036,15 @@ and the following Resource level attributes:
   Implementations MUST include `none` as one of the possible values and when
   set to `none` then compatibility checking MUST NOT be performed.
 
-  If the `enforcecompatibility` capability is set to `true` then
+  If the `compatibilityauthority` attribute is set to `server` then
   implementations of this specification are REQUIRED to perform the proper
   compatibility checks to ensure that all Versions of a Resource adhere to the
-  rules defined by the current value of this attribute. This includes
-  generating an error in response to requests to add/delete/modify Versions or
-  update this attribute in such a way as to make the existing Versions
-  incompatible due to the new value.
+  rules defined by the current value of this attribute.
+  For `compatibility` strategies that require understanding the sequence in
+  which Versions need to be compatible, the server MUST use the [`ancestor`]
+  (#ancestor-attribute) to determine the sequence of Versions. For Versions
+  that have the `ancestor` value set to its `versionid` attribute,
+  compatibility cannot be enforced.
```

Comment:
This last sentence is missing something... 1) it only focuses on **backwards** and leaves **forwards** out of the picture, 2) "cannot be enforced" sounds like some kind of bad situation, so perhaps it might be better to say something like "implicitly passes the checks due to there not being an ancestor". Of course, that again only focuses on backwards.

Seems to me we either need to enhance that sentence to be more abstract:
- when the previous, or next, Version in the hierarchy is needed to determine adherence to the compatibility rules, if none exists then the implied result of the validation is "true"

or, we can just drop the entire sentence because it's kind of obvious and the definition of the rules SHOULD tell people that "no parent/child means it passes".

Reply:
Good catch. I tried multiple options and ended up dropping the last sentence, mostly because it gets too much into the crux of the individual compatibility strategies where above we mention that the spec doesn't really dictate what those mean. Although it's unlikely that backward and forward change, I think it might read a bit clunky in the context of their transitive options as they mean compatibly across all parents/descendants and then only the next version is not sufficient.